### PR TITLE
Upgrade framework version(s) from 5.1 to 5.5!

### PR DESCRIPTION
**Pull request by DFurnes**
Well, this is a real heck of a diff. 😨 This pull request upgrades Aurora to Laravel 5.5, following the steps in each of the relevant upgrade guides. I've also updated the [spatie/laravel-backup](https://github.com/spatie/laravel-backup) package & removed a few packages that were incompatible with the framework upgrades (and no longer needed).

### CODE REVIEW

File: app/Providers/RouteServiceProvider.php [hunk 1]
```diff
@@ -20,45 +19,55 @@ class RouteServiceProvider extends ServiceProvider
...
+        $this->mapApiRoutes();
 
-            if (! $key) {
-                throw new NotFoundHttpException;
-            }
+        $this->mapWebRoutes();
 
-            return $key;
-        });
+        //
```
Comment (itsjoekent) [inline app/Providers/RouteServiceProvider.php]: 🔨

File: app/Providers/RouteServiceProvider.php [hunk 2]
```diff
@@ -20,45 +19,55 @@ class RouteServiceProvider extends ServiceProvider
...
     {
-        $northstar = gateway('northstar');
-
-        $router->bind('users', function ($id) use ($northstar) {
-            $user = $northstar->getUser('id', $id);
-
-            if (! $user) {
-                throw new NotFoundHttpException;
-            }
+        //
```
Comment (itsjoekent) [inline app/Providers/RouteServiceProvider.php]: another one!

File: app/Providers/EventServiceProvider.php [hunk 3]
```diff
@@ -13,20 +13,19 @@ class EventServiceProvider extends ServiceProvider
...
-     * @param  \Illuminate\Contracts\Events\Dispatcher  $events
      * @return void
      */
-    public function boot(DispatcherContract $events)
+    public function boot()
     {
-        parent::boot($events);
+        parent::boot();
 
         //
```
Comment (itsjoekent) [inline app/Providers/EventServiceProvider.php]: ....and another one!

File: app/Providers/AuthServiceProvider.php [hunk 4]
```diff
@@ -15,13 +14,14 @@ class AuthServiceProvider extends ServiceProvider
...
-     * @param  \Illuminate\Contracts\Auth\Access\Gate  $gate
      * @return void
      */
-    public function boot(GateContract $gate)
+    public function boot()
     {
-        parent::registerPolicies($gate);
+        $this->registerPolicies();
+
+        //
```
Comment (itsjoekent) [inline app/Providers/AuthServiceProvider.php]: did you forget to vacuum

File: app/Providers/RouteServiceProvider.php (hunk 1 above)
Comment (DFurnes) [reply app/Providers/RouteServiceProvider.php]: Haha, these are included with the framework as "placeholders". 😄

Comment (itsjoekent) [reply app/Providers/RouteServiceProvider.php]: oh

Comment (itsjoekent) [reply app/Providers/RouteServiceProvider.php]: BRB

**Comment by DFurnes** [comment]
You're forgiven! It's a doozy. 🌀

**Pull request merged**